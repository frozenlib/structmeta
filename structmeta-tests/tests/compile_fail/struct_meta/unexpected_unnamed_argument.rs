#[structmeta_tests::attr_optional_name_value(10)]
fn my_func() {}

fn main() {}
