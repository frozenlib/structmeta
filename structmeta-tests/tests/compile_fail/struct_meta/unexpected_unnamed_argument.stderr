error: too many unnamed arguments.
 --> $DIR/unexpected_unnamed_argument.rs:1:46
  |
1 | #[structmeta_tests::attr_optional_name_value(10)]
  |                                              ^^
